File: Dr.NutrizioNino.Models/Dto/FoodDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dr.NutrizioNino.Models.Dto;

public record FoodDto(
    Guid Id
    , string Name
    , decimal Quantity
    , string? Barcode
    , Guid? BrandId
    , IList<NutrientDto> Nutrients
    )
{ }

File: src/Dr.NutrizioNino.WebVue/src/Interfaces/foods/FoodDto.ts
import type { FoodNutrientDto } from './FoodNutrientDto'

export interface FoodDto {
  id: string
  name: string
  quantity: number
  unitOfMeasureId: string | null
  barcode: string | null
  brandId: string | null
  nutrients: FoodNutrientDto[]
  supermarketIds: string[] | null
  categoryIds: string[] | null
}
